Sources/SpotHeroAPI/Search/Monthly/Models/MonthlyRate.swift

```diff
@@ -18,8 +19,11 @@ public struct MonthlyRate: Codable {
     /// Monthly parking amenities offered for this rate at the facility.
     public let amenities: [Amenity]
     
-    /// Information concerning the redemption process for customers who park at a monthly facility.
-    public let redemption: Redemption<MonthlyRedemptionInstructions>
+    /// Information concerning the redemption process for customers who park at a facility.
+    public let redemptionInstructions: MonthlyRedemptionInstructions
+    
+    /// Defines the type of parking available at a garage.
```

Comment:
`type of parking ` makes me think of the commonly used enum `parkingType` throughout SpotHero code, which refers to `transient`, `airport`, etc.

What about:
```suggestion
    /// Defines the garage's reservation redemption type.
```